src/pages/articles.tsx

```diff
@@ -233,10 +257,10 @@ const Articles = ({ data: { site, allMdx } }) => {
   )
 }
 
-export default Articles
+export default ArticlesPage
 
 export const pageQuery = graphql`
-  query {
+  query ArticlesPage {
```

Comment:
All the queries should be named to have a respective name in graphql codegen generated typescript file

Reply:
makes sense!